tutorials/quick_start/keras_fw/quant.py

```diff
@@ -66,12 +90,51 @@ def quantize(model: tf.keras.Model,
         batch_size=int(args[BATCH_SIZE])
     )
 
-    core_config = mct.core.CoreConfig(quantization_config=mct.core.QuantizationConfig(shift_negative_activation_correction=True))
-
-    quantized_model, quantization_info = \
-        mct.ptq.keras_post_training_quantization_experimental(model,
-                                                              representative_data_gen,
-                                                              core_config=core_config,
-                                                              target_platform_capabilities=tpc)
-
-    return quantized_model, QuantInfo(user_info=quantization_info, tpc_info=tpc.get_info(), technique='PTQ')
+    # Mixed-precision configurations
+    if args.get('mp_weights_compression', False):
+        mp_conf = MixedPrecisionQuantizationConfigV2()
+        core_conf = CoreConfig(mixed_precision_config=mp_conf)
+        mp_wcr = args['mp_weights_compression']
+        target_kpi = get_target_kpi(model, mp_wcr, representative_data_gen, core_conf, tpc)
+    else:
+        core_conf = CoreConfig()
+        mp_wcr = None
+        target_kpi = None
+
+    # Quantize model
+    if args.get('gptq', False):
+
+        technique = 'GPTQ'
+        n_epochs = args.get('gptq_num_calibration_iter') // n_iter
+        logging.info(
+            f"MCT Gradient-based Post Training Quantization is enabled. Number of epochs: {n_epochs}")
+
+        gptq_representative_data_gen = get_representative_dataset(
+            representative_dataset_folder=args[REPRESENTATIVE_DATASET_FOLDER],
+            n_iter=n_iter,
+            batch_size=int(args[BATCH_SIZE])
+        )
+
+        gptq_conf = mct.gptq.get_keras_gptq_config(n_epochs=n_epochs)
+
+
+        quantized_model, quantization_info = \
+            mct.gptq.keras_gradient_post_training_quantization_experimental(model,
+                                                                              representative_data_gen=representative_data_gen,
+                                                                              target_kpi=target_kpi,
+                                                                              core_config=core_conf,
+                                                                              gptq_config=gptq_conf,
+                                                                              gptq_representative_data_gen=gptq_representative_data_gen,
+                                                                              target_platform_capabilities=tpc)
+
+
+    else:
+        technique = 'PTQ'
+        quantized_model, quantization_info = \
+            mct.ptq.keras_post_training_quantization_experimental(model,
```

Comment:
once you remove _gptq_representative_data_gen_ you can call the API outside the if-else